### ISSUE TITLE
With the back space first element is removed 

### ISSUE THREAD

**Issue by krishbhattacharyya** [orthes/medium-editor-insert-plugin#442]
Hi,
Thank you for this beautiful plugin. 
This is not an issue, I have only one question regarding this plugin. I have downloaded your repository files & noticed, the first element "h2" tag is removed with the back space but in live site it is still appears with the back space. I need like the live site, where the first element should be there.

http://linkesch.com/medium-editor-insert-plugin/

Basically this heading "jQuery insert plugin for MediumEditor", can not be removed. 

Is there any option configuration for stop this thing or I have to change code?
I just want to know this. 

Waiting for reply.

**Comment by krishbhattacharyya** [comment]
One more thing I have noticed today **this is a bug** in firefox it is removed but in the google chrome it works fine.

http://linkesch.com/medium-editor-insert-plugin/

"jQuery insert plugin for MediumEditor"- after using back space.

**Comment by krishbhattacharyya** [comment]
Sorry this is not an issue. I am closing this. Actually this is related to my custom addons.
